app/assets/javascripts/misc/previous-address.js

```diff
@@ -1,4 +1,6 @@
 import 'classlist.js';
+import { TextField } from 'field-kit';
```

Comment:
They allow you to cherry-pick specific methods out of a larger library.  In this case I just need the`TextField` portion of the `field-kit` library.